### ISSUE TITLE
Toggled menu items do not announce the toggle state to screen readers

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab#17501]
> Menu items where icons indicate toggle state do not have `title` not `aria-label` to tell users that this entry is toggled/enabled
>   ![image](https://github.com/user-attachments/assets/f67eea37-ecde-4b9e-b766-ab1835b43e47)
>   ![image](https://github.com/user-attachments/assets/b16904fd-6b7e-4c1f-84a5-764d99258738)
>
> _Originally posted by @krassowski in https://github.com/jupyterlab/jupyterlab/pull/17472#pullrequestreview-2775324935_

**Comment by nkn2022** [comment]
@krassowski This has been fixed. We could close this now.
